docs/cookbook/creating_landing_page_blocks_(enterprise).md

```diff
@@ -59,25 +24,34 @@ For example:
 
 ``` php
 <?php
-namespace AcmeDemoBundle\Block;
+namespace AcmeBundle\Block;
 
 use EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Model\AbstractBlockType;
 
-/**
-* RSS block
-* Renders feed from a given URL.
-*/
-class RSSBlock extends AbstractBlockType
```

Comment:
A real-life example in a Cookbook is what I am waiting for. So why change this RSSBlock by a generic ExampleBlock?

Reply:
Because RSSBlock is built in into the application. So 1: it's already available as an example, 2: the user cannot add this code and experiment with it, because it's already in the app.

Reply:
I see 2 ideas here:
1/ either we explain an existing built-in solution as something you can reproduce and custom for your needs.
2/ or we find another example for this Recipe, from real-world (not generic)

Reply:
As I wrote in a comment below, I kinda agree and think we should later move this page and "Creating landing page layouts" to Content rendering in Guide - there a generic instruction will be in place, and the information will be also easier to find